samples/cpp/benchmark_app/utils.cpp

```diff
@@ -122,31 +121,51 @@ std::vector<std::string> parse_devices(const std::string& device_string) {
         return std::vector<std::string>();
 
     auto devices = split(comma_separated_devices, ',');
-    return devices;
+    result.insert(result.end(), devices.begin(), devices.end());
+    return result;
 }
 
 std::map<std::string, std::string> parse_value_per_device(const std::vector<std::string>& devices,
                                                           const std::string& values_string) {
     //  Format: <device1>:<value1>,<device2>:<value2> or just <value>
     std::map<std::string, std::string> result;
     auto device_value_strings = split(values_string, ',');
+    auto if_auto = std::find(devices.begin(), devices.end(), "AUTO") != devices.end();
```

Comment:
Can we avoid `MULTI`/`AUTO` plugin specific code in common function which just doing simple parsing?

Reply:
Actions for MULTI/AUTO plugin has been moved to another function.